editor/gui/editor_file_dialog.cpp

```diff
@@ -1826,11 +1839,15 @@ EditorFileDialog::EditorFileDialog() {
 	makedir->connect("pressed", callable_mp(this, &EditorFileDialog::_make_dir));
 	pathhb->add_child(makedir);
 
-	list_hb = memnew(HSplitContainer);
+	body_hbox = memnew(HBoxContainer);
+	body_hbox->set_v_size_flags(Control::SIZE_EXPAND_FILL);
 
 	vbc->add_child(pathhb);
-	vbc->add_child(list_hb);
-	list_hb->set_v_size_flags(Control::SIZE_EXPAND_FILL);
+	vbc->add_child(body_hbox);
```

Comment:
I know it's existing code, but it's weird that we put this unrelated line in between creating and adding the container. `vbc->add_child(pathhb);` should be moved above `body_hbox = memnew(HBoxContainer);` (with a line break).